jihyunkim-api/models/signupDao.js

```diff
@@ -0,0 +1,43 @@
+/* eslint-disable no-unused-vars */
+import prisma from '../prisma';
+import bcrypt from 'bcrypt';
+
+const userSignUp = async (req) => {
+  const { email, name, password } = req.body;
+
+  if (!email || !password) {
+    const error = new Error('필수 정보를 입력해주세요 ☄️ ');
+    error.statusCode = 400;
+    throw error;
+  }
```

Comment:
```suggestion
  if (!email || !password) {
    const error = new Error('KEY_ERROR');
    error.statusCode = 400;
    throw error;
  }
```
이렇게 꼭 들어와야하는 키가 들어오지 않은 경우를 보고
키에러라고 합니다 :) 시도 good!

그런데 이런 로직도 Dao에 있으면 안됩니다.
`req`가 제대로 되었는지 검증하는 것은 컨트롤러에서 진행하시는 것이 맞습니담

Reply:
리뷰해주신 사항 반영하여 수정 완료했습니다! 감사합니다 소헌님 🙇‍♀️ fd41d21443deac4052b89dfe236ebf116eac2fc0